app/screens/search/search.js

```diff
@@ -140,6 +141,16 @@ export default class Search extends PureComponent {
         }
     }
 
+    navigationButtonPressed({buttonId}) {
+        if (buttonId === 'backPress') {
+            if (this.state.preview) {
+                this.refs.preview.handleClose();
+            } else {
+                this.props.navigator.dismissModal();
```

Comment:
are we still using the navigator here? is this going to be done in a later PR? when we do, lets make sure this is actually needed.